### ISSUE TITLE
Extend data collected on who is using the Design System

### ISSUE THREAD

**Issue by trang-erskine** [alphagov/govuk-design-system#1768]
## What

Include data on local authorities and the other 43% of GOV.UK services that we don't know about yet. 
We also need to make this process repeatable (at least quarterly). This may require procuring a tool to do this more easily. 

## Why
Follow on from the previous work Sara did. 

## Who needs to know about this
Sara, Trang, Devs

## Done when

- [ ] Thing to do

**Comment by vanitabarrett** [comment]
@trang-erskine Should this be an epic card?

**Comment by trang-erskine** [comment]
Yes. I flipped flopped on this a bit. This work should include running the 90,000 LA services through wappalyzer which we attempted to, but it didn't work out. This might also include added something in the code to make it easier to wappalyzer (or another tool) to pick up which framework is being used. Maybe even the which version of Frontend the service is using.
